src/mgr/DaemonServer.cc

```diff
@@ -1998,6 +1998,17 @@ bool DaemonServer::handle_command(MCommand *m)
     }
     return true;
   } else {
+    if (!pgmap_ready) {
+      derr << "Warning: due to ceph-mgr restart, some PG states may not be up to date" << dendl;
+      ss << "Warning: due to ceph-mgr restart, some PG states may not be up to date\n";
```

Comment:
why do you put a newline at the end of the warning message? and i am inclined to not include "Warning:" in this message. as `ss` is supposed to include the error messages.

Reply:
If we do not add "\n", it will become the following print(connected with ‘dumped pgs_brief ’):

>[root@cephfs102 ~]# ceph pg dump pgs_brief    
>Warning: due to ceph-mgr restart, some PG states may not be up to date dumped pgs_brief 

'ss' output to the cli interface, there is no redundant error message identification, such as printing information above, if not, increase the "Warning:" can better prompt the user?